notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>nit - are these bullets rendering properly in markdown? I feel like they might not be</p><p> </p><p>Love the note about the privacy concerns and thank you for linking to the info.</p><p> </p><p> </p><p>With the trip distance and basing it on Manhattan being 13.4 miles long - just to double check, are you completely ignoring Brooklyn? This use case is also only looking at citibike stations in manhattan, right? (if not, that might help your model! :) )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>I changed the bullet sign from <code>-</code> to <code>*</code> . Do they look different?</p><p>In this project, only data in Manhattan being used. Both the Taxi and the Bike datasets are filtered that trips must start and end in Manhattan.</p>